sdk/python/packages/flet-core/src/flet_core/date_picker.py

```diff
@@ -0,0 +1,342 @@
+from datetime import datetime, date
+from enum import Enum
+from typing import Any, Optional, Union
+
+from flet_core.control import Control, OptionalNumber
+from flet_core.ref import Ref
+from flet_core.text_style import TextStyle
+from flet_core.types import (
+    ResponsiveNumber,
+)
+
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal
+
+KeyboardTypeString = Literal[
+    None,
+    "text",
+    "multiline",
+    "number",
+    "phone",
+    "datetime",
+    "email",
+    "url",
+    "visiblePassword",
+    "name",
+    "streetAddress",
+    "none",
+]
+
+
+class KeyboardType(Enum):
+    NONE = "none"
+    TEXT = "text"
+    MULTILINE = "multiline"
+    NUMBER = "number"
+    PHONE = "phone"
+    DATETIME = "datetime"
+    EMAIL = "email"
+    URL = "url"
+    VISIBLE_PASSWORD = "visiblePassword"
+    NAME = "name"
+    STREET_ADDRESS = "streetAddress"
+
+
+DatePickerModeString = Literal[
+    "day",
+    "year"
+]
+
+
+class DatePickerMode(Enum):
```

Comment:
Let's expose both `DatePickerMode` and `DatePickerEntryMode` to `flet_core/__init__.py` to avoid using `import flet_core` in user's code.